### ISSUE TITLE
Deleting pod that had mount a host directory, does not let a new pod mount that directory

### ISSUE THREAD

**Issue by cn-mayank** [kubernetes/community#2614]
/sig-windows

**Kubernetes version** 1.10.0

I have a deployment on my single worker-node cluster that mounts a directory. If the pod is deleted and when another pod gets created on the same node, it is not able to mount the directory with error `<path> is not a directory` even though the path exists, is a directory and was mounted by the pod before.

It is a reproducible issue for me, and i have verified it is not a docker issue as i manually created a docker container with a volume mount. killed it and new manual docker container was able to mount the same directory.

It could be that pod is not releasing some resource properly when dying

**Comment by tianshapjq** [comment]
you may locate the wrong place to issue your problem, please ref to [kubernetes/kubernetes](https://github.com/kubernetes/kubernetes)

**Comment by cn-mayank** [comment]
@krmayankk @tianshapjq I went through the contributing guidelines as you rightly pointed out, my previous understanding about it was incorrect.
Here is my current understanding,
1. First reach out on slack/mail (I had done that previously but didn't get any response)
2. I am confused about posting on stack-overflow vs opening an github issue. 
3. Was sig windows the wrong place and Would /sig-storage be the right place ? I raised with sig-windows as this issue seemed to affect only on windows node.

Appreciate your feedback. Thanks!
